taglib/ogg/xiphcomment.cpp

```diff
@@ -99,6 +99,17 @@ String Ogg::XiphComment::album() const
   return d->fieldListMap["ALBUM"].toString();
 }
 
+String Ogg::XiphComment::albumArtist() const
+{
+  if(!d->fieldListMap["ALBUMARTIST"].isEmpty())
```

Comment:
`ALBUMARTIST` isn't in the [Xiph spec](https://www.xiph.org/vorbis/doc/v-comment.html#fieldnames), but `PERFORMER` is. Why `ALBUMARTIST`?